__tests__/endpoints/groups.test.js

```diff
@@ -1,28 +1,19 @@
-import connect from 'mongo.js'
 import request from 'supertest'
 import app from 'app.js'
 
-jest.mock('mongo.js')
-
 describe('GET /groups', () => {
-  const mockDistinct = jest.fn().mockResolvedValue([
-'stanford',
-'pcc'
-])
-  const mockCollection = jest.fn().mockReturnValue({distinct: mockDistinct})
-  const mockDb = {collection: mockCollection}
-  connect.mockReturnValue(mockDb)
 
   it('returns the resource', async () => {
 
     const res = await request(app)
       .get('/groups')
       .set('Accept', 'application/json')
     expect(res.statusCode).toEqual(200)
-    expect(res.body).toEqual({data: [
-{id: 'stanford'},
-{id: 'pcc'}
-]})
-    expect(mockDistinct.mock.calls[0][0]).toBe('group')
-  })
+    expect(res.type).toEqual('application/json')
+    expect(res.body.data).toEqual(expect.arrayContaining([
+      {id: "stanford", "label": "Stanford University"},
+      {id: "cornell", "label": "Cornell University"}
```

Comment:
looks for a couple example groups in the response to verify it works and the data structure looks ok, don't really see the need to replicate the whole list though